QGCCommon.pri

```diff
@@ -26,6 +26,7 @@ linux {
         message("Linux build")
         CONFIG  += LinuxBuild
         DEFINES += __STDC_LIMIT_MACROS
+        DEFINES += QGC_GST_AIRLINK_ENABLED
```

Comment:
I'm still seeing these in the code?

Reply:
Oh. I thought that was one of the requirements to make this as plugin so it could be disabled. I'll get rid of it.